docs/csharp/programming-guide/classes-and-structs/snippets/destructors/expr-bodied-destructor.cs

```diff
@@ -13,9 +13,10 @@ class Program
 {
    static void Main()
    {
-      var destroyer = new Destroyer();
-      destroyer = null;
-      GC.Collect();
-      Console.WriteLine("Exiting...");
-   }
+        var destroyer = new Destroyer();
+        destroyer = null;
+        GC.Collect();
+        GC.WaitForPendingFinalizers();
```

Comment:
💡 `GC.GetTotalMemory` typically performs this operation in a manner that better meets user expectations.

```suggestion
        GC.GetTotalMemory(forceFullCollection: true);
```